pkg/cli/cli.go

```diff
@@ -198,6 +198,11 @@ func (c *cli) initialize() error {
 		return fmt.Errorf("failed to read config: %v", err)
 	}
 
+	if projectConfig.IsV1() {
```

Comment:
This will panic if `err != nil`. Move into the `err == nil` case above, or:
```suggestion
	if c.projectVersion == config.Version1 {
```

Reply:
Done . Good catcher.